src/Analyzer.Utilities/AbstractVersionCheckAnalyzer.cs

```diff
@@ -12,7 +12,7 @@ namespace Analyzer.Utilities
     public abstract class AbstractVersionCheckAnalyzer : DiagnosticAnalyzer
     {
         private const string RuleId = "CA9999";
-        private const string AnalyzerPackageVersion = "2.3.*";
+        private const string AnalyzerPackageVersion = "2.5.*";
```

Comment:
AnalyzerPackageVersion used in this file is just for the diagnostic message. The actual version checking is done in AnalysisContextExtensions.